services/tenant-ui/frontend/src/components/verifier/PresentationDetails.vue

```diff
@@ -16,6 +16,21 @@
         <li>Contact Alias: {{ presentation.contact.alias }}</li>
         <hr />
       </div>
+
+      <!-- VERIFIED Meaning-->
+      <div v-if="props.show_information && presentation.status == 'verified'">
+        <span class="pi pi-check"></span
+        ><span
+          >Credential is held by
+          <strong>{{ presentation.contact.alias }}</strong></span
+        ><br />
+        <span class="pi pi-check"></span><span>Credential is valid</span><br />
+        <span class="pi pi-check"></span><span>Credential is tamper-free </span
+        ><br />
+        <span class="pi pi-check"></span
+        ><span>All attribute restrictions were statisfied</span><br />
+      </div>
```

Comment:
Kind of a wierd format with these span braces here. This our format settings causing it? I've seen the old vue extension do this and can clean it up by putting some manual newlines between sibling elements.

Reply:
the formatter is doing that. is there a better way to do this than span tags and  <br/> tags?

Reply:
No I think the tags are fine, it's just the line length when the tags are next to each other without a space can cause a wierd line break I think